com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -676,13 +676,16 @@ def parallel(self, width, routing=Routing.ROUND_ROBIN, func=None):
             func: Optional function called when :py:const:`Routing.HASH_PARTITIONED` routing is specified.
                 The function provides an integer value to be used as the hash that determines
                 the tuple channel routing.
+            name (String): The name to display for the parallel region.
 
         Returns:
             Stream: A stream for which subsequent transformations will be executed in parallel.
 
         """
+        if name is None:
+            name = "parallel"
```

Comment:
Personally, I think the default of `parallel` is more clear. Do you have a preference?